src/tls/cockpit-certificate-ensure.c

```diff
@@ -343,7 +368,16 @@ cockpit_certificate_find (CertificateKeyPair *result,
       return false;
     }
 
-  certificate_and_key_read (result, certificate_filename);
+  if (certificate_and_key_read (result, certificate_filename) < 0)
+    {
+      if (verbose)
+        printf ("Unable to read certificate file or key file\n");
```

Comment:
This is an error message, so it should go to stderr. But moreover, it loses useful information -- this needs to be written where the error actually happens, to include the `errno` error message and file name/path etc. Isn't that already supposed to happen with the `warn()` calls?

In other words, this should just go away?

Reply:
I was trying to mimic the previous check (line 366: `printf ("Unable to find ...`).

But you are right, the previous `warn` should be enough. I will remove this.